modin/pandas/test/test_series.py

```diff
@@ -3570,8 +3421,12 @@ def test_tolist(data):
 
 
 @pytest.mark.parametrize("data", test_data_values, ids=test_data_keys)
-@pytest.mark.parametrize("func", agg_func_values, ids=agg_func_keys)
```

Comment:
Most functions are not classified as transformational; there is little point in testing exceptions in most cases here.